accounts/basic.go

```diff
@@ -28,7 +30,7 @@ func (ba BasicAuth) Validate(md MetaData) (string, error) {
 
 	if len(auth) > 0 {
 		user, password, ok := parseBasicAuth(auth[0])
-		fmt.Printf("User: %s Password: %s OK: %s\n", user, password, ok)
+		log.Infof("User: %s Password: %s OK: %t\n", user, password, ok)
```

Comment:
We should probably start censoring the password

Reply:
this has been since updated here https://github.com/bmeg/grip/blob/feature/indexing/accounts/basic.go#L32-L37